templates/config.edn

```diff
@@ -1,137 +1,167 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ :preferred-format "Markdown"
```

Comment:
source:  
 

```clojure
(defn get-preferred-format
  ([]
   (get-preferred-format (get-current-repo)))
  ([repo-url]
   (keyword
     (or
       (when-let [fmt (:preferred-format (get-config repo-url))]
         (string/lower-case (name fmt)))

       (get-in @state [:me :preferred_format] "markdown")))))
```

Reply:
@logseq-cldwalker should all be commented out? my only concern is that when users modify options that are available via the settings UI it will create a duplicate entry at the bottom of the file. 

Possible options as I see it:
A. comment out all options after setting the default value
B. uncomment all options w/ default value
C. uncomment-only options configurable via settings UI.

Reply:
It's worth noting that once the user changes the setting from the UI a new line will be added at the bottom of the file. The format selection is available in the settings UI:  
 

![](https://user-images.githubusercontent.com/25513724/235818811-ebd68509-48cb-490c-b9dd-9b9efa16ad95.png)